docs/Configuration.md

```diff
@@ -1,40 +1,220 @@
+[filelogreceiver_readme]: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/v0.27.0/receiver/filelogreceiver
+
+### Fluent Forward Receiver
+
+Fluent Forward Receiver runs a TCP server that accepts events via the [Fluent Forward
+protocol][fluent_forward_protocol].
+
+The basic configuration for Fluent Forward Receiver has following format:
+
+```yaml
+receivers:
+  fluentforward:
+    endpoint: 0.0.0.0:8006
+```
 
-Link to [Filelog Receiver detailed configuration][filelogreceiver_readme]
+for details please see [Fluent Forward Receiver documentation][fluentforwardreceiver_readme].
+
+[fluent_forward_protocol]: https://github.com/fluent/fluentd/wiki/Forward-Protocol-Specification-v1
+[fluentforwardreceiver_readme]: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/release/v0.27.x/receiver/fluentforwardreceiver
 
 ### Syslog Receiver
 
-Syslog Receiver simple config
+Syslog Receiver parses Syslogs from tcp/udp using
+the [opentelemetry-log-collection](https://github.com/open-telemetry/opentelemetry-log-collection) library.
+
+The basic example for Syslog Receiver with TCP configuration:
+
+```yaml
+receivers:
+  syslog:
+    tcp:
+      listen_address: "0.0.0.0:54526"
+    protocol: rfc5424
+```
+
+The basic example for Syslog Receiver with UDP Configuration:
+
+```yaml
+receivers:
+  syslog:
+    udp:
+      listen_address: "0.0.0.0:54526"
+    protocol: rfc3164
+    location: UTC
+```
+
+for details please see [Syslog Receiver documentation][syslogreceiver_readme].
+
+[syslogreceiver_readme]: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/v0.27.0/receiver/syslogreceiver
+
+### Statsd Receiver
+
+StatsD Receiver ingests [StatsD messages][statsd_messages] into the OpenTelemetry Collector.
 
-Link to [Syslog Receiver detailed configuration][syslogreceiver_readme]
+The basic configuration for StatsD Receiver has following format:
+
+```yaml
+receivers:
+  statsd:
+  statsd/2:
+    endpoint: "localhost:8127"
+    aggregation_interval: 70s
+    enable_metric_type: true
+    timer_histogram_mapping:
+      - statsd_type: "histogram"
+        observer_type: "gauge"
+      - statsd_type: "timing"
+        observer_type: "gauge"
+```
+
+for details please see [StatsD Receiver documentation][statsdreceiver_readme].
+
+[statsd_messages]: https://github.com/statsd/statsd/blob/master/docs/metric_types.md
+[statsdreceiver_readme]: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/release/v0.27.x/receiver/statsdreceiver
 
 ### Telegraf Receiver
 
-Telegraf Receiver simple config
+Telegraf Receiver ingests metrics from various [input plugins][input_plugins]
+into OTC pipeline.
+
+The basic configuration for Telegraf Receiver has following format:
+
+```yaml
+receivers:
+  telegraf:
+    separate_field: false
+    agent_config: |
+      [agent]
+        interval = "2s"
+        flush_interval = "3s"
```

Comment:
I'm not 100% sure about this but I'd set those to the same value.

Reply:
set to the same value